### PR TITLE
STU-218 Adding Block support for LeCroy Oscilloscope T3DSO1000(A)-2000

### PR DESCRIPTION
**Pull request by LatentDream**
image

Demo

### CODE REVIEW

**Comment by LatentDream** [comment]
@dstrande, Am I missing an important type of block for a scope ?
- And could this one be a value-add, in your opinion [Delay measurement between channel](https://cdn.teledynelecroy.com/files/manuals/tds031000-2000_programming_manual.pdf#page=98) ?

**Comment by jackparmer** [comment]
@dstrande pls review so we can merge

File: blocks/HARDWARE/OSCILLOSCOPES/TELEDYNE_LECROY/T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from flojoy import flojoy, DataContainer, VisaConnection
+from typing import Literal, Optional
+
+
+@flojoy(inject_connection=True)
+def DISPLAY_ON_OFF_T3DSO1XXX(
+    connection: VisaConnection,
+    ch1: Literal["ON", "OFF"] = "ON",
```
Comment (dstrande) [inline blocks/HARDWARE/OSCILLOSCOPES/TELEDYNE_LECROY/T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX.py:8]: These types of parameters with 2 selections should probably be boolean, I think, from a user perspective. With True being ON for example.

**Review by dstrande — COMMENTED** [review]
Looks good. Can always add the delay measurements or other blocks later.

File: blocks/HARDWARE/OSCILLOSCOPES/TELEDYNE_LECROY/T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX.py (hunk 1 above)
Comment (LatentDream) [reply blocks/HARDWARE/OSCILLOSCOPES/TELEDYNE_LECROY/T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX.py]: New UX:
image

**Comment by LatentDream** [comment]
@dstrande if you have some time today, can you review the change, thanks

**Review by dstrande — APPROVED** [review]

**Pull request merged**